client/main.js

```diff
@@ -1,177 +1,226 @@
-import Vue from 'vue'
-import Router from 'vue-router'
-import infiniteScroll from 'vue-infinite-scroll'
-import vueSelect from 'vue-select'
-import vueModal from 'vue-js-modal'
-import vueSplit from 'vue-split-panel'
-import qs from 'friendly-querystring'
-import http from './http'
-import moment from 'moment'
-import promiseFinally from 'promise.prototype.finally'
-
-import DateRangePicker from './widgets/date-range-picker.vue'
-import detailList from './widgets/detail-list.vue'
-import barLoader from './widgets/bar-loader.vue'
-import dataViewer from './widgets/data-viewer.vue'
-import copyButton from './widgets/copy.vue'
-
-import snapscroll from './directives/snapscroll'
-
-import App from './App.vue'
-import Intro from './routes/Intro.vue'
-import Workflows from './routes/Workflows.vue'
-import DomainConfig from './routes/domain-config.vue'
-import ExecutionTabs from './routes/execution/index.vue'
-import ExecutionSummary from './routes/execution/summary.vue'
-import History from './routes/execution/history.vue'
-import StackTrace from './routes/execution/stack-trace.vue'
-import Queries from './routes/execution/queries.vue'
-import TaskList from './routes/task-list.vue'
+import Vue from 'vue';
```

Comment:
probably worth running eslint --fix (with eslint-plugin-prettier) on all files to minimize future diff noise like this

Reply:
Yea good idea, I was thinking to raise a separate PR with eslint run. I'm not sure eslint has been correctly setup for this project but will try to raise it for next PR. Thanks!

Reply:
Branch setup with these eslint fixes. Will hopefully raise a PR today. Thanks.
https://github.com/uber/cadence-web/tree/eslint-fixes